src/views/studio/UploadEditVideoActionSheet/UploadEditVideoActionSheet.style.tsx

```diff
@@ -0,0 +1,158 @@
+  height: var(--upload-video-action-sheet-bar-height);
+  border-bottom: solid 1px ${colors.gray[700]};
+`
+
+export const TabsContainer = styled.div`
+  display: grid;
+  grid-auto-flow: column;
+  grid-auto-columns: max-content;
+  scrollbar-width: thin;
+  overflow: auto hidden;
+  align-items: center;
+
+  *:first-of-type {
+    margin: 0px 8px;
+  }
+`
+
+export const ButtonsContainer = styled.div`
+  display: grid;
+  align-items: center;
+  grid-auto-flow: column;
+  grid-gap: ${sizes(1)};
+  padding: 0 ${sizes(3)};
+  border-left: solid 1px ${colors.gray[700]};
+
+  @media screen and (min-width: ${breakpoints.small}) {
+    grid-gap: ${sizes(4)};
+    padding: 0 ${sizes(6)};
+  }
+`
+
+export const Content = styled.div<{ height: number }>`
+  ${StudioContainerStyle}
+  display: grid;
+  grid-template-rows: max-content max-content;
+  grid-template-columns: 100%;
+  height: ${({ height }) => height}px;
+  overflow-y: auto;
+
+  @media screen and (min-width: ${breakpoints.medium}) {
+    height: initial;
+    grid-gap: ${sizes(12)};
+    grid-template-rows: 1fr;
+    grid-template-columns: 1fr 1fr;
+  }
+`
+
+export const StyledCheckboxContainer = styled.div`
+  display: flex;
+  margin-bottom: 32px;
+  p {
+    margin-left: 20px;
+  }
+`
+
+export const StyledRadioContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`
+
+export const FileDropperContainer = styled.div`
+  padding: ${sizes(8)} ${sizes(4)} 0 ${sizes(4)};
+  @media screen and (min-width: ${breakpoints.medium}) {
+    padding: ${sizes(8)} 0 ${sizes(8)} ${sizes(8)};
+  }
+`
+
+export const FormContainer = styled.form<{ height: number }>`
+  display: grid;
+  grid-auto-flow: row;
+  padding: ${sizes(12)} ${sizes(4)} ${sizes(8)} ${sizes(4)};
+
+  @media screen and (min-width: ${breakpoints.medium}) {
+    overflow-y: auto;
+    height: ${({ height }) => height}px;
+    padding: ${sizes(8)} ${sizes(24)} ${sizes(8)} 8px;
+  }
+
+  /* title */
+  > div:first-of-type {
+    margin-bottom: ${sizes(4)};
+  }
+
+  textarea {
+    height: 3em;
+    @media screen and (min-width: ${breakpoints.small}) {
+      height: initial;
+    }
+  }
+`
+
+export const Tab = styled.div<{ selected: boolean }>`
+  height: 100%;
+  max-width: 178px;
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: space-around;
+  padding: 0 0 0 ${sizes(3)};
+  align-items: center;
+  cursor: pointer;
+  user-select: none;
+  ${({ selected }) => selected && `border-bottom: 3px solid ${colors.blue[500]};`}
+  > button {
+    margin-left: ${sizes(1)};
+  }
+`
+
+export const TabTitle = styled(Text)`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`
+
+export const DrawerOverlay = styled(animated.div)`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: ${colors.transparentBlack[66]};
+  pointer-events: none;
+`
+
+export const plusIconStyle = css`
+  width: initial;
+  height: initial;
```

Comment:
The button icon styles has a fixed width and height of 16px which is not ideal because if the icon is not of that size weird things can happen. In this case, the Icon is 14px width and height and doesn't scale so it looks like it's not centered.

this makes it so the Icon can use its initial width and height of 14px instead of 16px imposed by the component